docs/guides/latest-updates.mdx

```diff
@@ -124,17 +124,20 @@ Users can purchase licenses for the following functions contributed by our partn
 To get started, explore the [Qiskit Functions documentation](/guides/functions).
 
 
-## IBM Quantum blog: IBM Quantum delivers on performance challenge
+## IBM Quantum blog: Two new blogs in January
 
 *Browse all blogs at the [IBM Quantum blog page](https://www.ibm.com/quantum/blog).*
 
 {/* Bring over the Quantum news (blog) announcement post */}
 
-Two years ago, IBM set a challenge for both itself and the quantum community. We asked quantum researchers to work on developing quantum algorithms incorporating circuits with 100 qubits and gate depths of 100. In exchange, we would build a quantum computer capable of returning accurate values for those circuits in less than a day’s runtime.
+The first IBM Quantum blog post of the year takes a look back at some of the amazing tools released in 2024 that help make quantum software development easier and more accessible than ever before.
+The blog provides recaps of major releases from last year, including Qiskit v1.x, Qiskit addons, Qiskit Transpiler Service, and more - all with code examples and documentation links to help readers get started using them.
```

Comment:
Are these supposed to the same paragraph?

Reply:
yes.